apps/data-types.md

```diff
@@ -176,6 +176,13 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | internalNote | string | Internal note for the customer done by the merchant. |
 | billingAddress | [address](page:apps-data-types#address) | The billing address of the customer. |
 
+## deliveryWeightQuantity
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| amount | number | The amount displayed as a decimal number. |
+| unit | enum | The unit displayed as abbreviated unit, if available. Can be *g*, *kg*, *mg*, *oz*, *lb* or *t*. Otherwise a localised name of the unit is displayed. Can be gram(s), *kilogram(s)*, *milligram(s)*, *ounce(s)*, *pound(s)* or *ton(s)*.  |
```

Comment:
I don't know any case when the abbreviation is not available. we should not confuse the people and the description should just be:

Can be *g*, *kg*, *mg*, *oz*, *lb* or *t*